home_workshop_misc/init.lua

```diff
@@ -96,7 +96,7 @@ else
 end
 
 local MODPATH = minetest.get_modpath("home_workshop_misc")
-dofile(MODPATH.."/crafts.lua")
+if minetest.get_modpath("default") and minetest.get_modpath("basic_materials") then dofile(MODPATH.."/crafts.lua") end
```

Comment:
Should be on multiple lines.
```suggestion
if minetest.get_modpath("default") and minetest.get_modpath("basic_materials") then
	dofile(MODPATH.."/crafts.lua")
end
```

Reply:
this is a pointless loc increase for a simple thing under luachecks default line length limit

Reply:
Nothing to do with line length, all to do with readability.

And readability also rules over loc.

Reply:
still readable, but sure, small enough. (waste not want not)
https://github.com/mt-mods/home_workshop_modpack/pull/3/commits/1ce05336722d120641ce4dfb64e19c8478b6abb7 resolved

Reply:
Another personal opinion here. I'd like multi line style when there's single if+dofile and single line style when there's multiple packed together (when it forms clean large block for loading files).

Basically two styles I'd prefer personally depending on actual content:

```lua
local MODPATH = minetest.get_modpath("home_workshop_misc")

if minetest.get_modpath("default") then
	dofile(MODPATH.."/crafts.lua")
end

more_code()
```

and

```lua
local MODPATH = minetest.get_modpath("home_workshop_misc")

if minetest.get_modpath("default1") then dofile(MODPATH.."/crafts1.lua") end
if minetest.get_modpath("default2") then dofile(MODPATH.."/crafts2.lua") end
if minetest.get_modpath("default3") then dofile(MODPATH.."/crafts3.lua") end
if minetest.get_modpath("default4") then dofile(MODPATH.."/crafts4.lua") end

more_code()
```

But personally I would avoid this (or same with additional line breaks between blocks):

```lua
local MODPATH = minetest.get_modpath("home_workshop_misc")

if minetest.get_modpath("default1") then
	dofile(MODPATH.."/crafts.lua1")
end
if minetest.get_modpath("default2") then
	dofile(MODPATH.."/crafts.lua2")
end
if minetest.get_modpath("default3") then
	dofile(MODPATH.."/crafts.lua3")
end
if minetest.get_modpath("default4") then
	dofile(MODPATH.."/crafts.lua4")
end

more_code()
```

I do agree with _"And readability also rules over loc."_ but then also in my opinion this specific case is bit too minor to really affect readability a lot and is mostly opinion based especially as long as there's no strict formatting rules that everyone can agree with and therefore should not affect final decision about merging PR.